code/game/gamemodes/cult/cult_comms.dm

```diff
@@ -0,0 +1,46 @@
+/datum/action/cultcomm
+	name = "Communion"
+	button_icon_state = "cult_comms"
+	background_icon_state = "bg_cult"
+	action_type = AB_INNATE
+	check_flags = AB_CHECK_RESTRAINED|AB_CHECK_STUNNED|AB_CHECK_LYING
```

Comment:
I took this from TG..i could make an EMERGENCY OH SHIT comm feature for when restrained...